src/configuration.rs

```diff
@@ -46,9 +46,9 @@ pub enum WhenConditionOperator {
     EqualOperator,
     #[serde(rename = "neq")]
     NotEqualOperator,
-    #[serde(rename = "starts_with")]
+    #[serde(rename = "startsWith")]
```

Comment:
```suggestion
    #[serde(rename = "startswith")]
```

😬

Reply:
already pushed the changes :)

Reply:
oh, you mean `startswith`, no camel case?

Reply:
ok ok... fixing it